pages/volunteers_new.tsx

```diff
@@ -1,16 +1,261 @@
+                  fontSize="large"
+                />
+              }
+              description="You are a seasoned professional, capable of leading yourself and others in your discipline."
+            />
+            <CardOne
+              title="Availability"
+              icon={
+                <EventAvailableOutlined
+                  sx={{ color: palette.text.secondary }}
+                  fontSize="large"
+                />
+              }
+              description="You dedicate the time you need to accomplish your committed tasks with Open Seattle."
+            />
+          </CardRowContainer>
+        </Stack>
+      </Container>
+      {/* Oath & Values Section */}
+      <Container
+        sx={{ backgroundColor: 'white' }}
+        maxWidth={false}
+        disableGutters
+      >
+        <Stack
+          gap={8}
+          sx={{
+            textAlign: 'center',
+            paddingY: 8,
+            maxWidth: 'md',
+            marginX: 'auto',
+          }}
+        >
+          <Typography variant="headlineLarge">Our Oath</Typography>
+          <Typography variant="bodyLarge">
+            We champion these values and ask you as a volunteer to adopt them,
+            too.
+          </Typography>
+          <Box sx={{ display: 'block' }}>
+            {oathContent.map((item, index) => (
+              <Accordion key={index}>
+                <AccordionSummary
+                  expandIcon={<AddOutlined />}
+                  aria-controls={`volunteer-values${index}-content`}
+                  id={`volunteer-values${index}-header`}
+                >
+                  <Typography variant="titleLarge">{item.label}</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography variant="bodyLarge">{item.content}</Typography>
+                </AccordionDetails>
+              </Accordion>
+            ))}
+          </Box>
+          <Link href="/volunteer-agreement" passHref>
+            <Button variant="contained" color="primary">
+              Read Our Volunteer Agreement
+            </Button>
+          </Link>
+        </Stack>
+      </Container>
+      {/* Process Section */}
+      <Container maxWidth={false}>
+        <Stack
+          gap={8}
+          sx={{
+            paddingY: 8,
+            maxWidth: 'md',
+            marginX: 'auto',
+          }}
+        >
+          <Typography variant="headlineLarge">The Process</Typography>
+          <ol>
+            {processContent.map((item, index) => (
+              <li key={index + 1}>
+                <Typography
+                  variant="titleLarge"
+                  color={palette.primary.main}
+                >{`${index + 1}.`}</Typography>
+                <Typography variant="bodyLarge" mx={2}>
+                  {item}
+                </Typography>
+              </li>
+            ))}
+          </ol>
+        </Stack>
+      </Container>
+      {/* Volunteer Application */}
```

Comment:
You can also break sections into components (optionally)

Reply:
Oh, that is a good idea. I'll try to implement that when I take care of the mobile version!